LKSH340/iocBoot/iocLKSH340-IOC-01/config.xml

```diff
@@ -8,6 +8,7 @@
     <macro name="BITS" pattern="^[0-9]$" description="Serial communication number of bits, defaults to 7." defaultValue="7" hasDefault="YES" />
     <macro name="PARITY" pattern="^(even)|(odd)|(none)$" description="Serial communication parity, defaults to odd." defaultValue="odd" hasDefault="YES" />
     <macro name="STOP" pattern="^[0-9]$" description="Serial communication stop bit, defaults to 1." defaultValue="1" hasDefault="YES" />
+    <macro name="EXCITATION_THRESHOLD_FILE" pattern="^.*" description="The file to get temperature thresholds and excitation values to set from." defaultValue="None.txt", hasDefault="Yes" />
```

Comment:
As mentioned on other PR I think separate this into two macros - one for filename, one for whether this functionality is switched on (default it to off)